modules/linear_boltzmann_solvers/discrete_ordinates_solver/sweep/fluds/aah_fluds_common_data.cc

```diff
@@ -60,12 +60,12 @@ AAH_FLUDSCommonData::InitializeAlphaElements(const SPDS& spds,
   std::set<int> location_boundary_dependency_set;
 
   // csoi = cell sweep order index
-  so_cell_inco_face_face_category_.reserve(spls.item_id.size());
-  so_cell_outb_face_slot_indices_.reserve(spls.item_id.size());
-  so_cell_outb_face_face_category_.reserve(spls.item_id.size());
-  for (int csoi = 0; csoi < spls.item_id.size(); ++csoi)
+  so_cell_inco_face_face_category_.reserve(spls.size());
+  so_cell_outb_face_slot_indices_.reserve(spls.size());
+  so_cell_outb_face_face_category_.reserve(spls.size());
+  for (int csoi = 0; csoi < spls.size(); ++csoi)
   {
-    int cell_local_id = spls.item_id[csoi];
+    int cell_local_id = spls[csoi];
```

Comment:
Please, change the type to `auto`. The for loops below can use that too. In general, I thinking, using `auto` is better...